introspection-engine/introspection-engine-tests/tests/tables/mod.rs

```diff
@@ -4,15 +4,21 @@ use introspection_engine_tests::{assert_eq_datamodels, test_api::*};
 use quaint::prelude::Queryable;
 use test_macros::test_each_connector_mssql as test_each_connector;
 
-#[test_each_connector]
+#[test_each_connector(tags("postgres", "mysql", "sqlite"))]
```

Comment:
Not testing MSSQL here?

Reply:
Ah, good spot, I forgot to revert that change